src/test/java/com/github/stefanbirkner/fakesftpserver/rule/FakeSftpServerRuleTest.java

```diff
@@ -38,6 +38,25 @@ public void SFTP_server_accepts_connections_with_password() {
     }
 
     @Test
+    public void SFTP_server_accepts_connections_with_random_port() {
+        FakeSftpServerRule sftpServer = new FakeSftpServerRule().withPort(0);
+        executeTestWithRule(() -> {
+            connectToServer(sftpServer);
+            assertThat(sftpServer.getPort()).isNotEqualTo(0);
+        }, sftpServer);
+    }
+
+    @Test
+    public void SFTP_server_accepts_connections_with_different_port() {
+        final int diffPort = 12345;
```

Comment:
Please inline `diffPort`. I know that this is my personal preference only, but I find it easier to read.